### PR TITLE
Grant permissions for onloadfs files of all classes

### PR DESCRIPTION
**Pull request by ghost**
The existing implementation of `fs_rw_onload_sockets()` grants permissions only to the `sock_file` class.  In recent versions of Onload (201606-u1 and later), onloadfs sockets do not set `S_IFSOCK` and so are classed as `file`, not `sock_file`.  Furthermore, onloadfs pipes are `fifo_file`.  This patch grants permissions to all of the relevant classes, including `sock_file` for backwards-compatibility.

### CODE REVIEW

File: policy/modules/kernel/filesystem.if [hunk 1]
```diff
@@ -7014,8 +7014,10 @@ interface(`fs_rw_onload_sockets',`
 		type onload_fs_t;
 	')
 
+	allow $1 onload_fs_t:file {ioctl read write getattr};
```
Comment (wrabcak) [inline policy/modules/kernel/filesystem.if]: Could you please change line 7017 to:
rw_files_pattern($1, onload_fs_t, onload_fs_t)

**Review by wrabcak — CHANGES_REQUESTED** [review]
Thanks for PR, could you please see my comments?

File: policy/modules/kernel/filesystem.if (hunk 1 above)
Comment (ghost) [reply policy/modules/kernel/filesystem.if]: Thanks for the quick response.  I've made the change in 10a61df.  I'll make an equivalent change in https://github.com/containers/container-selinux/pull/66.

**Review by wrabcak — APPROVED** [review]
LGTM.

**Comment by wrabcak** [comment]
Merged, 
Thanks!

**Pull request merged**